### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by Pontuspts**
Dev_Pontus

### CODE REVIEW

File: lab-exercise-1/PontusFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class PontusFeature {
+    public void feature() {
+        // this is a new feature
+    }
+}
```
Comment (Jack5hj) [inline lab-exercise-1/PontusFeature.java:5]: that need to improve

**Review by Pontuspts — COMMENTED** [review]
fixed it

**Review by NoodlesCupCake — COMMENTED** [review]
This content isn't well-written, please improve it.

**Review by NoodlesCupCake — COMMENTED** [review]
good

File: lab-exercise-1/PontusFeature.java (hunk 1 above)
Comment (KRoos555) [reply lab-exercise-1/PontusFeature.java]: This contents is not well-written. Please improve it.

File: lab-exercise-1/PontusFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class PontusFeature {
+    public void feature() {
+        // this is a new feature
```
Comment (Vvidsky) [inline lab-exercise-1/PontusFeature.java:3]: This content is not well-written. Please improve it.

File: lab-exercise-1/PontusFeature.java [hunk 3]
```diff
@@ -1,5 +1,7 @@
 public class PontusFeature {
     public void feature() {
         // this is a new feature
+        int x = 0;
+        x = x+1;
     }
-}
+}
```
Comment (Vvidsky) [inline lab-exercise-1/PontusFeature.java:7]: Great work

**Pull request merged**